trade-finance-manager-ui/templates/case/parties/void.njk

```diff
@@ -0,0 +1,47 @@
+{% import '../_macros/back-link.njk' as back %}
```

Comment:
can we rename the `void.njk` to something else? it looks like an error type of page